### PR TITLE
Updated previously required request headers to now be optional

### PR DESCRIPTION
**Pull request by hafowler**

### CODE REVIEW

File: api-reference/beta/api/bitlocker-list-recoverykeys.md [hunk 1]
```diff
@@ -64,8 +64,8 @@ The response might also contain an `odata.nextLink`, which you can use to page t
 |Name|Description|
 |:---|:---|
 |Authorization|Bearer {token}. Required.|
-|ocp-client-name|Name of the client application performing the API call. Required.|
-|ocp-client-version|Version of the client application performing the API call. Required.|
+|ocp-client-name|Name of the client application performing the API call used for debugging purposes. Optional.|
```
Comment (davidmu1) [inline api-reference/beta/api/bitlocker-list-recoverykeys.md:67]:
```suggestion
|ocp-client-name|The name of the client application performing the API call. This header is used for debugging purposes. Optional.|
```

File: api-reference/beta/api/bitlocker-list-recoverykeys.md [hunk 2]
```diff
@@ -64,8 +64,8 @@ The response might also contain an `odata.nextLink`, which you can use to page t
...
+|ocp-client-version|Version of the client application performing the API call used for debugging purposes. Optional.|
```
Comment (davidmu1) [inline api-reference/beta/api/bitlocker-list-recoverykeys.md:68]:
```suggestion
|ocp-client-version|The version of the client application performing the API call. This header is used for debugging purposes. Optional.|
```

File: api-reference/beta/api/bitlockerrecoverykey-get.md [hunk 3]
```diff
@@ -60,8 +60,8 @@ This method supports the `$select` OData query parameter to return the **key** p
 |Name|Description|
 |:---|:---|
 |Authorization|Bearer {token}. Required.|
-|ocp-client-name|Name of the client application performing the API call. Required.|
-|ocp-client-version|Version of the client application performing the API call. Required.|
+|ocp-client-name|Name of the client application performing the API call used for debugging purposes. Optional.|
```
Comment (davidmu1) [inline api-reference/beta/api/bitlockerrecoverykey-get.md:63]:
```suggestion
|ocp-client-name|The name of the client application performing the API call. This header is used for debugging purposes. Optional.|
```

File: api-reference/beta/api/bitlockerrecoverykey-get.md [hunk 4]
```diff
@@ -60,8 +60,8 @@ This method supports the `$select` OData query parameter to return the **key** p
...
+|ocp-client-version|Version of the client application performing the API call used for debugging purposes. Optional.|
```
Comment (davidmu1) [inline api-reference/beta/api/bitlockerrecoverykey-get.md:64]:
```suggestion
|ocp-client-version|The version of the client application performing the API call. This header is used for debugging purposes. Optional.|
```

**Review by davidmu1 — APPROVED** [review]

**Comment by FaithOmbongi** [comment]
@hafowler Following up on this PR and it's related GitHub issue https://github.com/microsoftgraph/microsoft-graph-docs/issues/11041. Could you address the feedback so we can publish this info and close/resolve the customer issue? Thanks.

**Comment by hafowler** [comment]
@FaithOmbongi We are preparing to move this documentation to v1. I am going to close this PR, and create a new one to move the docs to v1 and include the suggested changes by @davidmu1

**Comment by hafowler** [comment]
@FaithOmbongi I have addressed feedback. Please let me know if there is anything else I can do.

**Review by FaithOmbongi — APPROVED** [review]
Minor changelog update request.

**Comment by hafowler** [comment]
@FaithOmbongi committed your suggestions

**Comment by FaithOmbongi** [comment]
@hafowler , you can set the **ready to merge** label when you're ready for your changes to go live.

**Comment by FaithOmbongi** [comment]
@hafowler, kindly off this PR from merge. Engineering is still validating the changelog entry affecting the header properties.

**Comment by hafowler** [comment]
> @hafowler , you can set the **ready to merge** label when you're ready for your changes to go live.

Done

**Comment by hafowler** [comment]
> @hafowler, kindly off this PR from merge. Engineering is still validating the changelog entry affecting the header properties.

Sorry, @FaithOmbongi - I don't understand what this ask is? What do you mean by "off this PR from merge"?

**Pull request merged**